### PR TITLE
[SYCL] Disabling failing tests on Windows

### PR DESCRIPTION
**Pull request by alexbatashev**
Test failures are caused by driver regression.

### CODE REVIEW

File: SYCL/Basic/image/image_sample.cpp [hunk 1]
```diff
@@ -1,7 +1,8 @@
 // RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
 // RUN: %CPU_RUN_PLACEHOLDER %t.out
 // RUN: %GPU_RUN_PLACEHOLDER %t.out
-// UNSUPPORTED: cuda || hip
+// Temporarily disable test on Windows due to regressions in GPU driver.
+// UNSUPPORTED: cuda || hip, windows
```
Comment (vladimirlaz) [inline SYCL/Basic/image/image_sample.cpp:5]: Why it is unsupported?
I guess it should be XFAIL:level_zero

**Review by vladimirlaz — CHANGES_REQUESTED** [review]

**Review by AlexeySachkov — APPROVED** [review]
Ok with `Printf/double.cpp` temporary disabling on Windows

File: SYCL/Basic/image/image_sample.cpp (hunk 1 above)
Comment (alexbatashev) [reply SYCL/Basic/image/image_sample.cpp:5]: XFAIL is a poor test disabling practice. When the test is fixed, it'll fire up in every test run.

Comment (vladimirlaz) [reply SYCL/Basic/image/image_sample.cpp:5]: this is intended. Otherwise we may miss fix/unstable tets behavior.

Comment (kbobrovs) [reply SYCL/Basic/image/image_sample.cpp:5]: + 1 for XFAIL. XFAIL shows a problem in implementation which must be fixed, UNSUPPORTED means the test is not meant to be run at all on unsupported config.

**Comment by kbobrovs** [comment]
@bader, review comments have not been addressed AFAICS - there is still `UNSUPPORTED`. Is this going to be fixed in a separate PR?

File: SYCL/Basic/image/image_sample.cpp (hunk 1 above)
Comment (bader) [reply SYCL/Basic/image/image_sample.cpp:5]: I think the comment clearly explains the motivation to disable test on Windows. I agree with Alex that using UNSUPPORTED to disable failing tests in llvm-test-suite is less disruptive for CI system than XFAIL.
As of today, it's not possible to make some commits to intel/llvm and do not break CI checks if we use XFAIL mark.

**Comment by bader** [comment]
> Is this going to be fixed in a separate PR?

It's better ask @alexbatashev.

**Pull request merged**